boards/arm/samv7/same70-qmtech/include/board.h

```diff
@@ -46,11 +46,11 @@
  *   CPU clock: 120MHz
  *
  * There can be two on-board crystals.  However, the 32.768 crystal is not
```

Comment:
Please fix the text too. It is saying that "the 32.768 crystal in not populated on ..."